redhat-developer/developers.redhat.com
# [WIP] [RHDX-468] Fix node latest route

### JIRA Issue Link
https://projects.engineering.redhat.com/browse/RHDX-468

### Verification Process

The current state of this PR is to compare disabling trailing slash for a content type, and evaluate a possible change in editorial workflow.

- View a published Topic node as an editor
- See the local tasks block on the page
- Use the edit button
- Make a change to the node, save as Draft
- Upon save, you are taken to /my-node/latest (with no trailing slash)
- You are able to view your changes here, but /my-node displays the published state

Compare this workflow with other content types.

Before this can be merged, we need to discuss the intention of handling trailing slashes, or if we need a patch that the trailing_slash module does not interfere with workflow routes.
